src/util/workspace.ts

```diff
@@ -7,3 +8,17 @@ export function getWorkspaceFolderIfExists(): vscode.Uri | undefined {
   }
   return undefined;
 }
+
+// package name is all directories after the /java/ dir, otherwise empty
+export function getPackageName(srcDir: string): string {
+  let index = srcDir.toLowerCase().indexOf(path.sep + "java" + path.sep);
+  let defaultPackageName = "";
+  if (index > -1) {
+    // use everything after the "/java/" to be the package name
+    index = index + 6;
+    return (defaultPackageName = srcDir
```

Comment:
Strange it is both returning and assigning a value here. Should probably do one or the other.